.pre-commit-config.yaml

```diff
@@ -0,0 +1,39 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
@AgustinRamiroDiaz can we please also add conventional commits rules? https://www.conventionalcommits.org/en/v1.0.0/

Reply:
added!

Reply:
added!